app/views/transactions/index.html.haml

```diff
@@ -0,0 +1,21 @@
+-# TODO: app/views/accounts/transactions_in_review.html.haml is similar
+
+= render "shared/transactions/headers"
+
+- content_for :h1 do
+  = text("transactions.in_review")
+
+#header_billing.grid_9= render "shared/transactions/search"
```

Comment:
I think we may want to hide this search form if there are no `@order_details`. When that's empty, you can type in all the search fields, but there are no options.